src/test/kotlin/nl/info/client/brp/exception/BrpRuntimeExceptionTest.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * SPDX-FileCopyrightText: 2025 Lifely
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+
+package nl.info.client.brp.exception
+
+import io.kotest.core.spec.style.WordSpec
+import io.kotest.matchers.shouldBe
+import io.kotest.matchers.throwable.shouldHaveCauseOfType
+
+class BrpRuntimeExceptionTest : WordSpec({
```

Comment:
Huh? What is this doing in this PR?

And why a unit test to test something as simple as the instantiation of an exception class? I don't see this adding any value.

Reply:
had to increase coverage and chose something simple, as my pr was failing on dropping coverage :/

Reply:
The PR was not failing! It's just that codecov check that is failing. No big deal. You can merge your PR regardless. It's not a showstopper by any means.

I don't really like this unit test as it is not testing any meaningful logic so I would be in favour of removing it

Reply:
It's a shame that GitHub does not support the colour orange instead of red for 'warning checks' such as our codecov check. At least I am not aware of it. (https://github.com/isaacs/github/issues/1465)

Reply:
Huh, did I not see that properly?
Then I'll just roll-back these latest changes...